qdrant-landing/content/documentation/embeddings/gemini.md

```diff
@@ -38,52 +38,63 @@ The following example shows how to embed a document with the `models/embedding-0
 ## Embedding a document
 
 ```python
-import pathlib
-import google.generativeai as genai
-import qdrant_client
+import google.generativeai as gemini_client
+from qdrant_client import QdrantClient
+from qdrant_client.http.models import Distance, PointStruct, VectorParams
+collection_name = "example_collection"
 
 GEMINI_API_KEY = "YOUR GEMINI API KEY"  # add your key here
 
-genai.configure(api_key=GEMINI_API_KEY)
+gemini_client.configure(api_key=GEMINI_API_KEY)
+texts = [
+    "Qdrant is the best vector search engine!",
```

Comment:
It sounds a little bit posh to me

Reply:
maybe something like a good choice to be used with gemini (an example just to show the idea)

Reply:
Removed the posh example — but don't have something specific to show Gemini specific features